manifest.json

```diff
@@ -6,7 +6,7 @@
         "en": "AI-Powered Photos App for the Decentralized Web",
         "fr": "Gestion de photos en ligne"
     },
-    "version": "2022.28.05~ynh1",
+    "version": "2022.28.05~ynh2",
```

Comment:
When you write the version like that, wouldn’t it be an issue if we have to update for `2022.17.06` for example ? Will Yunohost correctly interpret that `2022.17.06` is an higher version than `2022.28.05 ` ?

Reply:
You are right ! 
I have followed the naming of the upstream app but I may have to change.